autoPlay.js

```diff
@@ -273,6 +273,7 @@ function goToLaneWithBestTarget() {
 				// ignore if lane is empty
 				if(g_Minigame.CurrentScene().m_rgGameData.lanes[i].dps == 0)
 					continue;
+					rainingGold = false;
```

Comment:
This line will never be reached because of the continue statement on the line above...